runner.py

```diff
@@ -44,9 +44,18 @@ def get_configs(path: Path) -> List[str]:
                         default='configs/config_example.json',
                         help='The path to a configuration file or '
                              'a directory that contains configuration files')
+    parser.add_argument('--device', '--devices', default='host cpu gpu None', type=str, nargs='+',
+                        choices=('host', 'cpu', 'gpu', 'None'),
```

Comment:
Minor: string
```python
"None"
```
looks confusing because of 
```python 
None
```
existance. May be, switch to `none`?